source/ranges.tex

```diff
@@ -6051,17 +6081,6 @@
 }
 \end{codeblock}
 
-\indexlibraryctor{common_view}%
-\begin{itemdecl}
-constexpr explicit common_view(V base);
-\end{itemdecl}
-
-\begin{itemdescr}
-\pnum
-\effects
-Initializes \exposid{base_} with \tcode{std::move(base)}.
-\end{itemdescr}
-
```

Comment:
This is the wrong constructor to remove. This one has to stay, the next one gets removed. See http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2020/p2236r0.html#3405